couchbase-entity/src/main/java/com/kaufland/generation/mapper/MapperGeneration.kt

```diff
@@ -95,27 +104,30 @@ class MapperGeneration {
                     .setter(field.setterFunSpec())
                     .build())
 
-            val resolverParam = ResolverParam()
-
 
-            resolverParam?.apply {
-                resolverParam.fromMapBuilder.beginControlFlow("map[%S]?.let", field.mapName)
-                resolverParam.toMapBuilder.beginControlFlow("obj.%N?.let", field.accessorName)
-                resolveDeclaringName(field.declaringName, resolverParam, field.accessorName, holder.typeParams)
-                resolverParam.fromMapBuilder.endControlFlow()
-                resolverParam.fromMapBuilder.beginControlFlow("?.apply")
-                resolverParam.fromMapBuilder.addStatement("obj.%N=this", field.accessorName)
-                resolverParam.fromMapBuilder.endControlFlow()
-                resolverParam.toMapBuilder.endControlFlow()
-                resolverParam.toMapBuilder.beginControlFlow("?.apply")
-                resolverParam.toMapBuilder.addStatement("map[%S]=this", field.mapName)
-                resolverParam.toMapBuilder.endControlFlow()
 
-                fromMap.addCode(resolverParam.fromMapBuilder.build())
-                toMap.addCode(resolverParam.toMapBuilder.build())
-            }
 
+            try {
+                val resolverParam = ResolverParam()
+                resolverParam?.apply {
```

Comment:
I don't think, we need the "?" here: we just created the object, so it cannot be null, right?